bark_ml/tests/model_loader_test.cc

```diff
@@ -0,0 +1,36 @@
+#include "libtensorflow/include/tensorflow/c/c_api.h"
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <string>
+#include <random>
+#include "bark_ml/model_loader/ModelLoader.hpp"
+
+
+int main()
+{
+    // NOTE: Modify this path in order to get the saves model
+    ModelLoader model("/Users/hart/Development/bark-ml/model/");
```

Comment:
relative paths

Reply:
what do you mean with relative path? The model is saved as stated in tfa.py locally and we used that same path here to get the model from it

Reply:
but why do you use my name?
a relative path would be `./model/`. need to create a bazel filgroup for that though

Reply:
actually it is the same style like in the current bark-ml/examples/tfa.py line43/44.
Why use different implementations?